oracle/datadog_checks/oracle/data/conf.yaml.example

```diff
@@ -56,6 +56,11 @@ instances:
     #
     password: <PASSWORD>
 
+    ## @param use_instant_client - boolean - optional - default: false
+    ## Determines whether the Agent uses the Oracle Instant Client to connect instead of the Oracle native client.
```

Comment:
```suggestion
    ## Determines whether the Agent uses the Oracle Instant Client to connect instead of the Oracle native client. This option cannot be used at the same time as `jdbc_driver_path`.
```